test/css.js

```diff
@@ -154,3 +154,12 @@ QUnit.test( "jQuery.cssNumber", function( assert ) {
 
 	assert.ok( jQuery.cssNumber, "jQuery.cssNumber exists" );
 } );
+
+QUnit.test( "jQuery.css with object as name and number as value", function( assert ) {
```

Comment:
Let's make it clear this is not public API, the second parameter should just be ignored:
```suggestion
QUnit.test( "An unsupported jQuery.fn.css(Object,Number) signature", function( assert ) {
```